lib/dotcom_web/views/trip_plan_view.ex

```diff
@@ -1,3 +1,4 @@
+# credo:disable-for-this-file
```

Comment:
📓 Disabling credo for this file because we're going to rewrite it...most likely moving this logic to a file in dotcom rather than dotcom_web since it has to do with domain logic rather than display.